### PR TITLE
Chore/ba 792 trx test data for trial version

### PR DESCRIPTION
**Pull request by hgo-ds**
Belongs to BA-807

### CODE REVIEW

File: tx-backend/src/main/resources/db/testdata-trial/afterMigrate__053_received_investigation_on_as_built_declined.sql [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+    (id                                      , contract_agreement_id, edc_url, notification_reference_id             , created_by     , send_to  , investigation_id           , target_date                          , severity          , created_by_name    , send_to_name , edc_notification_id                   , status    , created                              , updated                               , message_id                            , is_initial)
+values
+    (${investigationNotificationReceivedId4a}, null                 , null   , '8925f21f-09eb-4789-81fb-ec221e9e1561', ${bpnCustomer2}, ${bpnOwn}, ${investigationReceivedId4}, current_timestamp + interval '3 days', 'LIFE_THREATENING', ${bpnCustomer2Name}, ${bpnOwnName}, '8925f21f-09eb-4789-81fb-ec221e9e1561', 'DECLINED', current_timestamp - interval '4 days', current_timestamp - interval '2 hours', 'e04f75e8-d37b-42e4-8cf7-6127f35f3ed5', false);
+
+---
+-- join investigation to asset
+insert into assets_as_built_notifications
+    (notification_id                         , asset_id)
+values
+    (${investigationNotificationReceivedId4a}, ${assetAsBuiltId13});
```
Comment (felixschmidtcofinity) [inline tx-backend/src/main/resources/db/testdata-trial/afterMigrate__053_received_investigation_on_as_built_declined.sql:28]:
```suggestion
    (${investigationNotificationReceivedId4a}, ${assetAsBuiltId16});
```

File: tx-backend/src/main/resources/db/testdata-trial/afterMigrate__053_received_investigation_on_as_built_declined.sql [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+
+---
+-- join investigation to asset
+insert into assets_as_built_investigations
+    (investigation_id           , asset_id)
+values
+    (${investigationReceivedId4}, ${assetAsBuiltId13});
```
Comment (felixschmidtcofinity) [inline tx-backend/src/main/resources/db/testdata-trial/afterMigrate__053_received_investigation_on_as_built_declined.sql:35]:
```suggestion
    (${investigationReceivedId4}, ${assetAsBuiltId16});
```

File: tx-backend/src/main/resources/db/testdata-trial/afterMigrate__053_received_investigation_on_as_built_declined.sql [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+    (id                                      , contract_agreement_id, edc_url                    , notification_reference_id, created_by, send_to        , investigation_id           , target_date                          , severity          , created_by_name, send_to_name       , edc_notification_id                   , status    , created                              , updated                              , message_id                            , is_initial)
+values
+    (${investigationNotificationReceivedId4b}, 'contractAgreementId', 'http://localhost:5001/edc', null                     , ${bpnOwn} , ${bpnCustomer2}, ${investigationReceivedId4}, current_timestamp + interval '3 days', 'LIFE_THREATENING', ${bpnOwnName}  , ${bpnCustomer2Name}, '8925f21f-09eb-4789-81fb-ec221e9e1561', 'DECLINED', current_timestamp - interval '4 days', current_timestamp - interval '1 hour', '207ba6cf-217b-401d-a5da-69cac8b154a5', false);
+
+---
+-- join DECLINED notification to asset
+insert into assets_as_built_notifications
+    (notification_id                         , asset_id)
+values
+    (${investigationNotificationReceivedId4b}, ${assetAsBuiltId13});
```
Comment (felixschmidtcofinity) [inline tx-backend/src/main/resources/db/testdata-trial/afterMigrate__053_received_investigation_on_as_built_declined.sql:50]:
```suggestion
    (${investigationNotificationReceivedId4b}, ${assetAsBuiltId16});
```

File: tx-backend/src/main/resources/db/testdata-trial/afterMigrate__054_received_investigation_on_as_built_closed.sql [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+
+---
+-- reset sequence to highest next-val
+select setval('investigation_id_seq1', (select max(i.id) from investigation i), true);
+
+---
+insert into investigation_notification
+    (id                                      , contract_agreement_id, edc_url, notification_reference_id             , created_by     , send_to  , investigation_id           , target_date                           , severity, created_by_name    , send_to_name , edc_notification_id                   , status  , created                              , updated                               , message_id                            , is_initial)
+values
+    (${investigationNotificationReceivedId5a}, null                 , null   , '8925f21f-09eb-4789-81fb-ec221e9e1561', ${bpnCustomer2}, ${bpnOwn}, ${investigationReceivedId5}, current_timestamp + interval '1 month', 'MINOR' , ${bpnCustomer2Name}, ${bpnOwnName}, '8925f21f-09eb-4789-81fb-ec221e9e1561', 'CLOSED', current_timestamp - interval '5 days', current_timestamp - interval '2 hours', 'e04f75e8-d37b-42e4-8cf7-6127f35f3ed5', false);
```
Comment (felixschmidtcofinity) [inline tx-backend/src/main/resources/db/testdata-trial/afterMigrate__054_received_investigation_on_as_built_closed.sql:21]:
```suggestion
    (${investigationNotificationReceivedId5a}, null                 , null   , '8925f21f-09eb-4789-81fb-ec221e9e1561', ${bpnCustomer1}, ${bpnOwn}, ${investigationReceivedId5}, current_timestamp + interval '1 month', 'MINOR' , ${bpnCustomer1Name}, ${bpnOwnName}, '8925f21f-09eb-4789-81fb-ec221e9e1561', 'CLOSED', current_timestamp - interval '5 days', current_timestamp - interval '2 hours', 'e04f75e8-d37b-42e4-8cf7-6127f35f3ed5', false);
```

File: tx-backend/src/main/resources/db/testdata-trial/afterMigrate__054_received_investigation_on_as_built_closed.sql [hunk 5]
```diff
@@ -0,0 +1,50 @@
...
+-- (see https://documentation.red-gate.com/fd/callback-concept-184127466.html).
+-- it is only intended for usage in local or test environments
+
+-- This creates an investigation in state CLOSED in Severity Minor for asBuilt asset H-LeftFogLight which is sent from BPNL000CUSTOMER3 to BPNL000000000001
+
+---
+insert into investigation
+    (id                         , bpn            , close_reason                                 , created                              , description                                          , status  , side      , accept_reason, decline_reason, updated                               , error_message)
+values
+    (${investigationReceivedId5}, ${bpnCustomer2}, 'We confirm that the problem has been fixed.', current_timestamp - interval '5 days', 'Investigation on H-LeftFogLight due to malfunction.', 'CLOSED', 'RECEIVER', null         , null          , current_timestamp - interval '1 hours', null);
```
Comment (felixschmidtcofinity) [inline tx-backend/src/main/resources/db/testdata-trial/afterMigrate__054_received_investigation_on_as_built_closed.sql:11]:
```suggestion
    (${investigationReceivedId5}, ${bpnCustomer2}, 'We confirm that the problem has been fixed.', current_timestamp - interval '5 days', 'Investigation on Turn Lights due to malfunction.', 'CLOSED', 'RECEIVER', null         , null          , current_timestamp - interval '1 hours', null);
```

File: tx-backend/src/main/resources/db/testdata-trial/afterMigrate__054_received_investigation_on_as_built_closed.sql [hunk 6]
```diff
@@ -0,0 +1,50 @@
...
+    (notification_id                         , asset_id)
+values
+    (${investigationNotificationReceivedId5a}, ${assetAsBuiltId15});
+
+---
+-- join investigation to asset
+insert into assets_as_built_investigations
+    (investigation_id           , asset_id)
+values
+    (${investigationReceivedId5}, ${assetAsBuiltId15});
```
Comment (felixschmidtcofinity) [inline tx-backend/src/main/resources/db/testdata-trial/afterMigrate__054_received_investigation_on_as_built_closed.sql:35]:
```suggestion
    (${investigationReceivedId5}, ${assetAsBuiltId17});
```

File: tx-backend/src/main/resources/db/testdata-trial/afterMigrate__054_received_investigation_on_as_built_closed.sql [hunk 7]
```diff
@@ -0,0 +1,50 @@
...
+    (id                                      , contract_agreement_id, edc_url, notification_reference_id             , created_by     , send_to  , investigation_id           , target_date                           , severity, created_by_name    , send_to_name , edc_notification_id                   , status  , created                              , updated                               , message_id                            , is_initial)
+values
+    (${investigationNotificationReceivedId5a}, null                 , null   , '8925f21f-09eb-4789-81fb-ec221e9e1561', ${bpnCustomer2}, ${bpnOwn}, ${investigationReceivedId5}, current_timestamp + interval '1 month', 'MINOR' , ${bpnCustomer2Name}, ${bpnOwnName}, '8925f21f-09eb-4789-81fb-ec221e9e1561', 'CLOSED', current_timestamp - interval '5 days', current_timestamp - interval '2 hours', 'e04f75e8-d37b-42e4-8cf7-6127f35f3ed5', false);
+
+---
+-- join investigation to asset
+insert into assets_as_built_notifications
+    (notification_id                         , asset_id)
+values
+    (${investigationNotificationReceivedId5a}, ${assetAsBuiltId15});
```
Comment (felixschmidtcofinity) [inline tx-backend/src/main/resources/db/testdata-trial/afterMigrate__054_received_investigation_on_as_built_closed.sql:28]:
```suggestion
    (${investigationNotificationReceivedId5a}, ${assetAsBuiltId17});
```

File: tx-backend/src/main/resources/db/testdata-trial/afterMigrate__054_received_investigation_on_as_built_closed.sql [hunk 8]
```diff
@@ -0,0 +1,50 @@
...
+    (id                                      , contract_agreement_id, edc_url                    , notification_reference_id, created_by     , send_to  , investigation_id           , target_date                           , severity, created_by_name    , send_to_name , edc_notification_id                   , status  , created                               , updated                              , message_id                            , is_initial)
+values
+    (${investigationNotificationReceivedId5b}, 'contractAgreementId', 'http://localhost:5001/edc', null                     , ${bpnCustomer2}, ${bpnOwn}, ${investigationReceivedId5}, current_timestamp + interval '1 month', 'MINOR' , ${bpnCustomer2Name}, ${bpnOwnName}, '8925f21f-09eb-4789-81fb-ec221e9e1561', 'CLOSED', current_timestamp - interval '5 days ', current_timestamp - interval '1 hour', '207ba6cf-217b-401d-a5da-69cac8b154a5', false);
+
+---
+-- join CLOSED notification to asset
+insert into assets_as_built_notifications
+    (notification_id                         , asset_id)
+values
+    (${investigationNotificationReceivedId5b}, ${assetAsBuiltId15});
```
Comment (felixschmidtcofinity) [inline tx-backend/src/main/resources/db/testdata-trial/afterMigrate__054_received_investigation_on_as_built_closed.sql:50]:
```suggestion
    (${investigationNotificationReceivedId5b}, ${assetAsBuiltId17});
```

File: tx-backend/src/main/resources/db/testdata-trial/afterMigrate__054_received_investigation_on_as_built_closed.sql [hunk 9]
```diff
@@ -0,0 +1,50 @@
...
+values
+    (${investigationReceivedId5}, ${assetAsBuiltId15});
+
+---
+---
+-- CLOSED by sender notification message
+insert into investigation_notification
+    (id                                      , contract_agreement_id, edc_url                    , notification_reference_id, created_by     , send_to  , investigation_id           , target_date                           , severity, created_by_name    , send_to_name , edc_notification_id                   , status  , created                               , updated                              , message_id                            , is_initial)
+values
+    (${investigationNotificationReceivedId5b}, 'contractAgreementId', 'http://localhost:5001/edc', null                     , ${bpnCustomer2}, ${bpnOwn}, ${investigationReceivedId5}, current_timestamp + interval '1 month', 'MINOR' , ${bpnCustomer2Name}, ${bpnOwnName}, '8925f21f-09eb-4789-81fb-ec221e9e1561', 'CLOSED', current_timestamp - interval '5 days ', current_timestamp - interval '1 hour', '207ba6cf-217b-401d-a5da-69cac8b154a5', false);
```
Comment (felixschmidtcofinity) [inline tx-backend/src/main/resources/db/testdata-trial/afterMigrate__054_received_investigation_on_as_built_closed.sql:43]:
```suggestion
    (${investigationNotificationReceivedId5b}, 'contractAgreementId', 'http://localhost:5001/edc', null                     , ${bpnCustomer1}, ${bpnOwn}, ${investigationReceivedId5}, current_timestamp + interval '1 month', 'MINOR' , ${bpnCustomer1Name}, ${bpnOwnName}, '8925f21f-09eb-4789-81fb-ec221e9e1561', 'CLOSED', current_timestamp - interval '5 days ', current_timestamp - interval '1 hour', '207ba6cf-217b-401d-a5da-69cac8b154a5', false);
```

File: tx-backend/src/main/resources/db/testdata-trial/afterMigrate__060_sent_investigation_on_as_built_created.sql [hunk 10]
```diff
@@ -0,0 +1,35 @@
...
+-- (see https://documentation.red-gate.com/fd/callback-concept-184127466.html).
+-- it is only intended for usage in local or test environments
+
+-- This creates an investigation in state CREATED in Severity Critical for asBuilt asset Osram Front Left-AX400 which is sent from BPNL000000000001 to BPNL000SUPPLIER1
+
+---
+insert into investigation
+    (id                     , bpn      , close_reason, created                             , description                                                           , status   , side    , accept_reason, decline_reason, updated, error_message)
+values
+    (${investigationSentId1}, ${bpnOwn}, null        , current_timestamp - interval '1 day', 'Investigation on Osram Front Left-AX400 due to excessive brightness.', 'CREATED', 'SENDER', null         , null          , null   , null);
```
Comment (felixschmidtcofinity) [inline tx-backend/src/main/resources/db/testdata-trial/afterMigrate__060_sent_investigation_on_as_built_created.sql:11]:
```suggestion
    (${investigationSentId1}, ${bpnOwn}, null        , current_timestamp - interval '1 day', 'Investigation on Left Headbulb due to excessive brightness.', 'CREATED', 'SENDER', null         , null          , null   , null);
```

File: tx-backend/src/main/resources/db/testdata-trial/afterMigrate__061_sent_investigation_on_as_built_sent.sql [hunk 11]
```diff
@@ -0,0 +1,35 @@
...
+-- (see https://documentation.red-gate.com/fd/callback-concept-184127466.html).
+-- it is only intended for usage in local or test environments
+
+-- This creates an investigation in state SENT in Severity Critical for asBuilt asset Osram Front Right-AX400 which is sent from BPNL000000000001 to BPNL000SUPPLIER1
+
+---
+insert into investigation
+    (id                     , bpn      , close_reason, created                              , description                                                            , status, side    , accept_reason, decline_reason, updated                              , error_message)
+values
+    (${investigationSentId2}, ${bpnOwn}, null        , current_timestamp - interval '2 days', 'Investigation on Osram Front Right-AX400 due to excessive brightness.', 'SENT', 'SENDER', null         , null          , current_timestamp - interval '1 hour', null);
```
Comment (felixschmidtcofinity) [inline tx-backend/src/main/resources/db/testdata-trial/afterMigrate__061_sent_investigation_on_as_built_sent.sql:11]:
```suggestion
    (${investigationSentId2}, ${bpnOwn}, null        , current_timestamp - interval '2 days', 'Investigation on Right Headbulb due to excessive brightness.', 'SENT', 'SENDER', null         , null          , current_timestamp - interval '1 hour', null);
```

File: tx-backend/src/main/resources/db/testdata-trial/afterMigrate__062_sent_investigation_on_as_built_acknowledged.sql [hunk 12]
```diff
@@ -0,0 +1,49 @@
...
+-- (see https://documentation.red-gate.com/fd/callback-concept-184127466.html).
+-- it is only intended for usage in local or test environments
+
+-- This creates an investigation in state ACKNOWLEDGED in Severity Life-threatening for asBuilt asset Philips P-LeftHeadBulb which is sent from BPNL000000000001 to BPNL000SUPPLIER2
+
+---
+insert into investigation
+    (id                     , bpn      , close_reason, created                              , description                                                         , status        , side    , accept_reason, decline_reason, updated                              , error_message)
+values
+    (${investigationSentId3}, ${bpnOwn}, null        , current_timestamp - interval '3 days', 'Investigation on Philips P-LeftHeadBulb due to occasional outages.', 'ACKNOWLEDGED', 'SENDER', null         , null          , current_timestamp - interval '1 hour', null);
```
Comment (felixschmidtcofinity) [inline tx-backend/src/main/resources/db/testdata-trial/afterMigrate__062_sent_investigation_on_as_built_acknowledged.sql:11]:
```suggestion
    (${investigationSentId3}, ${bpnOwn}, null        , current_timestamp - interval '3 days', 'Investigation on Left Gland due to occasional outages.', 'ACKNOWLEDGED', 'SENDER', null         , null          , current_timestamp - interval '1 hour', null);
```

File: tx-backend/src/main/resources/db/testdata-trial/afterMigrate__063_sent_investigation_on_as_built_accepted.sql [hunk 13]
```diff
@@ -0,0 +1,49 @@
...
+-- (see https://documentation.red-gate.com/fd/callback-concept-184127466.html).
+-- it is only intended for usage in local or test environments
+
+-- This creates an investigation in state ACCEPTED in Severity Minor for asBuilt asset Würth W-FogLight which is sent from BPNL000000000001 to BPNL000SUPPLIER3
+
+---
+insert into investigation
+    (id                     , bpn      , close_reason, created                              , description                                                  , status    , side    , accept_reason         , decline_reason, updated                              , error_message)
+values
+    (${investigationSentId4}, ${bpnOwn}, null        , current_timestamp - interval '4 days', 'Investigation on Würth W-FogLight due to damaged packaging.', 'ACCEPTED', 'SENDER', 'Damn, it''s damaged!', null          , current_timestamp - interval '1 hour', null);
```
Comment (felixschmidtcofinity) [inline tx-backend/src/main/resources/db/testdata-trial/afterMigrate__063_sent_investigation_on_as_built_accepted.sql:11]:
```suggestion
    (${investigationSentId4}, ${bpnOwn}, null        , current_timestamp - interval '4 days', 'Investigation on Electric Fuse Small due to damaged packaging.', 'ACCEPTED', 'SENDER', 'Damn, it''s damaged!', null          , current_timestamp - interval '1 hour', null);
```

File: tx-backend/src/main/resources/db/testdata-trial/afterMigrate__063_sent_investigation_on_as_built_accepted.sql [hunk 14]
```diff
@@ -0,0 +1,49 @@
...
+    (id                                  , contract_agreement_id, edc_url, notification_reference_id, created_by, send_to        , investigation_id       , target_date                           , severity, created_by_name, send_to_name       , edc_notification_id                   , status    , created                              , updated                              , message_id, is_initial)
+values
+    (${investigationNotificationSentId4a}, 'contractAgreementId', null   , null                     , ${bpnOwn} , ${bpnSupplier3}, ${investigationSentId4}, current_timestamp + interval '1 month', 'MINOR' , ${bpnOwnName}  , ${bpnSupplier3Name}, '10454df4-c04d-4e48-ab86-6aeb568fe64a', 'ACCEPTED', current_timestamp - interval '4 days', current_timestamp - interval '1 hour', null      , true);
+
+---
+-- join investigation to asset
+insert into assets_as_built_notifications
+    (notification_id                     , asset_id)
+values
+    (${investigationNotificationSentId4a}, ${assetAsBuiltId22});
```
Comment (felixschmidtcofinity) [inline tx-backend/src/main/resources/db/testdata-trial/afterMigrate__063_sent_investigation_on_as_built_accepted.sql:28]:
```suggestion
    (${investigationNotificationSentId4a}, ${assetAsBuiltId07});
```

File: tx-backend/src/main/resources/db/testdata-trial/afterMigrate__063_sent_investigation_on_as_built_accepted.sql [hunk 15]
```diff
@@ -0,0 +1,49 @@
...
+
+---
+-- join investigation to asset
+insert into assets_as_built_investigations
+    (investigation_id       , asset_id)
+values
+    (${investigationSentId4}, ${assetAsBuiltId22});
```
Comment (felixschmidtcofinity) [inline tx-backend/src/main/resources/db/testdata-trial/afterMigrate__063_sent_investigation_on_as_built_accepted.sql:35]:
```suggestion
    (${investigationSentId4}, ${assetAsBuiltId07});
```

File: tx-backend/src/main/resources/db/testdata-trial/afterMigrate__063_sent_investigation_on_as_built_accepted.sql [hunk 16]
```diff
@@ -0,0 +1,49 @@
...
+    (id                                  , contract_agreement_id, edc_url, notification_reference_id             , created_by     , send_to  , investigation_id       , target_date                          , severity, created_by_name    , send_to_name , edc_notification_id                   , status    , created                              , updated, message_id, is_initial)
+values
+    (${investigationNotificationSentId4b}, 'contractAgreementId', null   , '10454df4-c04d-4e48-ab86-6aeb568fe64a', ${bpnSupplier3}, ${bpnOwn}, ${investigationSentId4}, current_timestamp + interval '1 week', 'MINOR' , ${bpnSupplier3Name}, ${bpnOwnName}, '10454df4-c04d-4e48-ab86-6aeb568fe64a', 'ACCEPTED', current_timestamp - interval '2 days', null   , null      , false);
+
+---
+-- join ACCEPTED notification to asset
+insert into assets_as_built_notifications
+    (notification_id                     , asset_id)
+values
+    (${investigationNotificationSentId4b}, ${assetAsBuiltId22});
```
Comment (felixschmidtcofinity) [inline tx-backend/src/main/resources/db/testdata-trial/afterMigrate__063_sent_investigation_on_as_built_accepted.sql:49]:
```suggestion
    (${investigationNotificationSentId4b}, ${assetAsBuiltId07});
```

File: tx-backend/src/main/resources/db/testdata-trial/afterMigrate__064_sent_investigation_on_as_built_declined.sql [hunk 17]
```diff
@@ -0,0 +1,49 @@
...
+-- (see https://documentation.red-gate.com/fd/callback-concept-184127466.html).
+-- it is only intended for usage in local or test environments
+
+-- This creates an investigation in state DECLINED in Severity Major for asBuilt asset Würth W-HighBeam which is sent from BPNL000000000001 to BPNL000SUPPLIER3
+
+---
+insert into investigation
+    (id                     , bpn      , close_reason, created                              , description                                             , status    , side    , accept_reason, decline_reason    , updated                              , error_message)
+values
+    (${investigationSentId5}, ${bpnOwn}, null        , current_timestamp - interval '5 days', 'Investigation on Würth W-HighBeam due to broken glass.', 'DECLINED', 'SENDER', null         , 'It''s not broken', current_timestamp - interval '1 hour', null);
```
Comment (felixschmidtcofinity) [inline tx-backend/src/main/resources/db/testdata-trial/afterMigrate__064_sent_investigation_on_as_built_declined.sql:11]:
```suggestion
    (${investigationSentId5}, ${bpnOwn}, null        , current_timestamp - interval '5 days', 'Investigation on Electric Fuse Big due to broken wire.', 'DECLINED', 'SENDER', null         , 'It''s not broken', current_timestamp - interval '1 hour', null);
```

File: tx-backend/src/main/resources/db/testdata-trial/afterMigrate__064_sent_investigation_on_as_built_declined.sql [hunk 18]
```diff
@@ -0,0 +1,49 @@
...
+    (id                                  , contract_agreement_id, edc_url                    , notification_reference_id, created_by, send_to        , investigation_id       , target_date                           , severity, created_by_name, send_to_name       , edc_notification_id                   , status    , created                              , updated                              , message_id                            , is_initial)
+values
+    (${investigationNotificationSentId5a}, 'contractAgreementId', 'http://localhost:5001/edc', null                     , ${bpnOwn} , ${bpnSupplier3}, ${investigationSentId5}, current_timestamp + interval '2 weeks', 'MAJOR' , ${bpnOwnName}  , ${bpnSupplier3Name}, '3ac2239a-e63f-4c19-b3b3-e6a2e5a240da', 'DECLINED', current_timestamp - interval '5 days', current_timestamp - interval '1 hour', '749b31e9-9e73-4699-9470-dbee67ebc7a7', true);
+
+---
+-- join investigation to asset
+insert into assets_as_built_notifications
+    (notification_id                     , asset_id)
+values
+    (${investigationNotificationSentId5a}, ${assetAsBuiltId22});
```
Comment (felixschmidtcofinity) [inline tx-backend/src/main/resources/db/testdata-trial/afterMigrate__064_sent_investigation_on_as_built_declined.sql:28]:
```suggestion
    (${investigationNotificationSentId5a}, ${assetAsBuiltId08});
```

File: tx-backend/src/main/resources/db/testdata-trial/afterMigrate__064_sent_investigation_on_as_built_declined.sql [hunk 19]
```diff
@@ -0,0 +1,49 @@
...
+
+---
+-- join investigation to asset
+insert into assets_as_built_investigations
+    (investigation_id       , asset_id)
+values
+    (${investigationSentId5}, ${assetAsBuiltId22});
```
Comment (felixschmidtcofinity) [inline tx-backend/src/main/resources/db/testdata-trial/afterMigrate__064_sent_investigation_on_as_built_declined.sql:35]:
```suggestion
    (${investigationSentId5}, ${assetAsBuiltId08});
```

File: tx-backend/src/main/resources/db/testdata-trial/afterMigrate__064_sent_investigation_on_as_built_declined.sql [hunk 20]
```diff
@@ -0,0 +1,49 @@
...
+    (id                                  , contract_agreement_id, edc_url                    , notification_reference_id, created_by     , send_to  , investigation_id       , target_date                           , severity, created_by_name    , send_to_name , edc_notification_id                   , status    , created                              , updated                              , message_id                            , is_initial)
+values
+    (${investigationNotificationSentId5b}, 'contractAgreementId', 'http://localhost:5001/edc', null                     , ${bpnSupplier3}, ${bpnOwn}, ${investigationSentId5}, current_timestamp + interval '2 weeks', 'MAJOR' , ${bpnSupplier3Name}, ${bpnOwnName}, '8925f21f-09eb-4789-81fb-ec221e9e1561', 'DECLINED', current_timestamp - interval '5 days', current_timestamp - interval '1 hour', '207ba6cf-217b-401d-a5da-69cac8b154a5', false);
+
+---
+-- join DECLINED notification to asset
+insert into assets_as_built_notifications
+    (notification_id                     , asset_id)
+values
+    (${investigationNotificationSentId5b}, ${assetAsBuiltId22});
```
Comment (felixschmidtcofinity) [inline tx-backend/src/main/resources/db/testdata-trial/afterMigrate__064_sent_investigation_on_as_built_declined.sql:49]:
```suggestion
    (${investigationNotificationSentId5b}, ${assetAsBuiltId08});
```

File: tx-backend/src/main/resources/db/testdata-trial/afterMigrate__065_sent_investigation_on_as_built_canceled.sql [hunk 21]
```diff
@@ -0,0 +1,35 @@
...
+
+---
+-- reset sequence to highest next-val
+select setval('investigation_id_seq1', (select max(i.id) from investigation i), true);
+
+---
+insert into investigation_notification
+    (id                                 , contract_agreement_id, edc_url                    , notification_reference_id, created_by, send_to        , investigation_id       , target_date                          , severity          , created_by_name, send_to_name       , edc_notification_id                   , status  , created                              , updated                              , message_id                            , is_initial)
+values
+    (${investigationNotificationSentId6}, 'contractAgreementId', 'http://localhost:5001/edc', null                     , ${bpnOwn} , ${bpnSupplier1}, ${investigationSentId6}, current_timestamp + interval '2 days', 'LIFE_THREATENING', ${bpnOwnName}  , ${bpnSupplier1Name}, '3ac2239a-e63f-4c19-b3b3-e6a2e5a240da', 'CLOSED', current_timestamp - interval '7 days', current_timestamp - interval '1 hour', '749b31e9-9e73-4699-9470-dbee67ebc7a7', true);
```
Comment (felixschmidtcofinity) [inline tx-backend/src/main/resources/db/testdata-trial/afterMigrate__065_sent_investigation_on_as_built_canceled.sql:21]:
```suggestion
    (${investigationNotificationSentId6}, 'contractAgreementId', 'http://localhost:5001/edc', null                     , ${bpnOwn} , ${bpnSupplier5}, ${investigationSentId6}, current_timestamp + interval '2 days', 'LIFE_THREATENING', ${bpnOwnName}  , ${bpnSupplier5Name}, '3ac2239a-e63f-4c19-b3b3-e6a2e5a240da', 'CLOSED', current_timestamp - interval '7 days', current_timestamp - interval '1 hour', '749b31e9-9e73-4699-9470-dbee67ebc7a7', true);
```

File: tx-backend/src/main/resources/db/testdata-trial/afterMigrate__065_sent_investigation_on_as_built_canceled.sql [hunk 22]
```diff
@@ -0,0 +1,35 @@
...
+-- (see https://documentation.red-gate.com/fd/callback-concept-184127466.html).
+-- it is only intended for usage in local or test environments
+
+-- This creates an investigation in state CANCELED in Severity Life-threatening for asBuilt asset Osram O-BrakeLight which is sent from BPNL000000000001 to BPNL000SUPPLIER1
+
+---
+insert into investigation
+    (id                     , bpn      , close_reason, created                              , description                                                , status    , side    , accept_reason, decline_reason, updated                              , error_message)
+values
+    (${investigationSentId6}, ${bpnOwn}, null        , current_timestamp - interval '7 days', 'Investigation on Osram O-BrakeLight due to strange noise.', 'CANCELED', 'SENDER', null         , null          , current_timestamp - interval '1 hour', null);
```
Comment (felixschmidtcofinity) [inline tx-backend/src/main/resources/db/testdata-trial/afterMigrate__065_sent_investigation_on_as_built_canceled.sql:11]:
```suggestion
    (${investigationSentId6}, ${bpnOwn}, null        , current_timestamp - interval '7 days', 'Investigation on LED Orange due to strange color.', 'CANCELED', 'SENDER', null         , null          , current_timestamp - interval '1 hour', null);
```

File: tx-backend/src/main/resources/db/testdata-trial/afterMigrate__065_sent_investigation_on_as_built_canceled.sql [hunk 23]
```diff
@@ -0,0 +1,35 @@
...
+    (id                                 , contract_agreement_id, edc_url                    , notification_reference_id, created_by, send_to        , investigation_id       , target_date                          , severity          , created_by_name, send_to_name       , edc_notification_id                   , status  , created                              , updated                              , message_id                            , is_initial)
+values
+    (${investigationNotificationSentId6}, 'contractAgreementId', 'http://localhost:5001/edc', null                     , ${bpnOwn} , ${bpnSupplier1}, ${investigationSentId6}, current_timestamp + interval '2 days', 'LIFE_THREATENING', ${bpnOwnName}  , ${bpnSupplier1Name}, '3ac2239a-e63f-4c19-b3b3-e6a2e5a240da', 'CLOSED', current_timestamp - interval '7 days', current_timestamp - interval '1 hour', '749b31e9-9e73-4699-9470-dbee67ebc7a7', true);
+
+---
+-- join investigation to asset
+insert into assets_as_built_notifications
+    (notification_id                    , asset_id)
+values
+    (${investigationNotificationSentId6}, ${assetAsBuiltId20});
```
Comment (felixschmidtcofinity) [inline tx-backend/src/main/resources/db/testdata-trial/afterMigrate__065_sent_investigation_on_as_built_canceled.sql:28]:
```suggestion
    (${investigationNotificationSentId6}, ${assetAsBuiltId12});
```

File: tx-backend/src/main/resources/db/testdata-trial/afterMigrate__065_sent_investigation_on_as_built_canceled.sql [hunk 24]
```diff
@@ -0,0 +1,35 @@
...
+
+---
+-- join investigation to asset
+insert into assets_as_built_investigations
+    (investigation_id       , asset_id)
+values
+    (${investigationSentId6}, ${assetAsBuiltId20});
```
Comment (felixschmidtcofinity) [inline tx-backend/src/main/resources/db/testdata-trial/afterMigrate__065_sent_investigation_on_as_built_canceled.sql:35]:
```suggestion
    (${investigationSentId6}, ${assetAsBuiltId12});
```

File: tx-backend/src/main/resources/db/testdata-trial/afterMigrate__066_sent_investigation_on_as_built_closed.sql [hunk 25]
```diff
@@ -0,0 +1,49 @@
...
+-- (see https://documentation.red-gate.com/fd/callback-concept-184127466.html).
+-- it is only intended for usage in local or test environments
+
+-- This creates an investigation in state CLOSED in Severity Critical for asBuilt asset Philips P-RightHeadBulb which is sent from BPNL000000000001 to BPNL000SUPPLIER2
+
+---
+insert into investigation
+    (id                     , bpn      , close_reason      , created                              , description                                                 , status  , side    , accept_reason, decline_reason, updated                              , error_message)
+values
+    (${investigationSentId7}, ${bpnOwn}, 'Problem is fixed', current_timestamp - interval '6 days', 'Investigation on Philips RightHeadBulb due to broken fuse.', 'CLOSED', 'SENDER', null         , null          , current_timestamp - interval '1 hour', null);
```
Comment (felixschmidtcofinity) [inline tx-backend/src/main/resources/db/testdata-trial/afterMigrate__066_sent_investigation_on_as_built_closed.sql:11]:
```suggestion
    (${investigationSentId7}, ${bpnOwn}, 'Problem is fixed', current_timestamp - interval '6 days', 'Investigation on Right Gland due to damaged package.', 'CLOSED', 'SENDER', null         , null          , current_timestamp - interval '1 hour', null);
```

**Review by felixschmidtcofinity — COMMENTED** [review]
Made some content and asset changes, for our demo dataset

**Review by rogocof — APPROVED** [review]
I reviewed and tested the new test data locally. It is very helpful for communicating issues externally. Thanks a lot!

**Pull request merged**